### PR TITLE
Documentation cesar

### LINKED ISSUE #116: [DOCUMENTATION] Quality attributes
# What section is going to be affected?
Quality scenarios

# What is missing?
We need to include a stimulus for each scenario.

### PR DESCRIPTION
**Pull request by themrcesi**
Documentation fixes.

Please, take a look at them and give me feedback.

### CODE REVIEW

**Comment by themrcesi** [comment]
Closes #116

**Comment by themrcesi** [comment]
Not ready to merge yet, quality tree diagram should be redone.

**Review by lamasumas — APPROVED** [review]
LGTM

File: src/docs/09_design_decisions.adoc [hunk 1]
```diff
@@ -4,7 +4,12 @@
 === Route specification
 We follow the https://github.com/Arquisoft/viadeSpec[Viade Specficiation] to set a format to manage the routes, so that interoperability can be achieved.
 
-
 === Leaflet
 We chose this library because it is open source, free, and matches the philosophy of Solid.
 
+=== Template
+At the begining of the course, we had to choose using the Inrupt Generator or not. After some discussion, we decided not to use it in order to learn React from the basics, as we considered not only important the SOLID challenge, but also learning a new technology.
```
Comment (alvarezGarciaMarcos) [inline src/docs/09_design_decisions.adoc]: I will maybe rephrase that and say something like: _we decided not to use it in order to learn React from the basics, because the main goal of this subject is learning how to design a software arquitecture, and with a template extracted from the internet you are force to use the one that is provided._ Just to remark our use of the plain framework.

File: src/docs/09_design_decisions.adoc [hunk 2]
```diff
@@ -4,7 +4,12 @@
...
+This decision carried us some disadvantages, for example we lost some time learning React, but we consider it worthy because of the knowledge gained.
+
+=== External Manual
+As it might be difficult for tech newbies the SOLID concepts or our application, we have created an external webpage that will work as a manual of our appllication. This was done in another webpage so the actual application was separated from this manual. This external manual will be hosted in https://lamasumas.github.io/Solid/[HERE].
```
Comment (alvarezGarciaMarcos) [inline src/docs/09_design_decisions.adoc]: You have a typo on the first line, _SOLID concepts **of** our application_. Also another typo on the second line: _that will work as a manual of our **application**._
